recipes/trouve/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "trouve" %}
+{% set version = "0.5.1" %}
+{% set author = "rwhitt2049" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: https://github.com/rwhitt2049/{{ name|lower }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - pandas
+    - toolz
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - pandas
+    - toolz
+
+test:
+  imports:
+    - trouve
+    
+extra:
+  recipe-maintainers:
+    - {{ author }}
+    
+about:
```

Comment:
In this section, just write explicit urls instead of using jinja templating variables